src/ray/raylet/lineage_cache.h

```diff
@@ -303,19 +301,13 @@ class LineageCache {
   /// was successful (whether we were subscribed).
   bool UnsubscribeTask(const TaskID &task_id);
 
-  /// The client ID, used to request notifications for specific tasks.
-  /// TODO(swang): Move the ClientID into the generic Table implementation.
-  ClientID client_id_;
-  /// The durable storage system for task information.
-  gcs::TableInterface<TaskID, TaskTableData> &task_storage_;
-  /// The pubsub storage system for task information. This can be used to
-  /// request notifications for the commit of a task entry.
-  gcs::PubsubInterface<TaskID> &task_pubsub_;
+  /// The handler for accessing task information from GCS.
+  gcs::TaskInfoAccessor &task_info_accessor_;
```

Comment:
Okay to hold a reference of `GcsClient` here.

Reply:
done